dango/account/margin/src/execute.rs

```diff
@@ -47,3 +66,166 @@ pub fn receive(_ctx: MutableCtx) -> StdResult<Response> {
     // Do nothing, accept all transfers.
     Ok(Response::new())
 }
+
+pub fn liquidate(ctx: MutableCtx, liquidation_denom: Denom) -> anyhow::Result<Response> {
+    let app_cfg: AppConfig = ctx.querier.query_app_config()?;
+
+    // Query account health
+    let HealthResponse {
+        total_debt_value,
+        utilization_rate,
+        total_collateral_value: _,
+        total_adjusted_collateral_value,
+        debts,
+        collaterals,
+    } = ctx
+        .querier
+        .query_health(ctx.contract, Some(ctx.funds.clone()))?;
+
+    // Ensure account is undercollateralized
+    ensure!(
+        utilization_rate > Udec128::ONE,
+        "account is not undercollateralized"
+    );
```

Comment:
Is the account liquidatable with utilization rate == 1?

This would be a super minor edge case but I'm curious how other protocols do this